modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -16,11 +16,15 @@ ifdef::shared-vpc[]
 = Adding the ingress DNS records
 endif::shared-vpc[]
 
-If you removed the DNS Zone configuration when creating Kubernetes manifests and
-generating Ignition configs, you must manually create DNS records that point at
-the ingress load balancer. You can create either a wildcard
-`*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other
-records per your requirements.
+DNS Zone configuration is removed when Kubernetes manifests and generates Ignition configs. You must manually create DNS records that point at the ingress load balancer. You can create either a wildcard
```

Comment:
It looks like in the rest of the assembly, it's mostly lowercase "zone" in "DNS zone". I do see the one uppercase instance in the Prerequisites list below here though, FWIW. Not sure which is accurate, but I would assume lowercase.

Reply:
I think the word "creating" might also be missing in the first sentence:

"DNS zone configuration is removed when creating Kubernetes manifests and generating Ignition configs."